src/java/com/sarality/app/view/dialog/CustomViewDialogFragment.java

```diff
@@ -33,14 +33,21 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
   public void onActivityCreated(Bundle savedInstanceState) {
     super.onActivityCreated(savedInstanceState);
     dataSource = createDataSource();
-    if (dataSource == null) {
-      render(null);
-    } else {
+    if (dataSource != null) {
```

Comment:
Adding this dataSource !=null test will break things when the dialog doesn't need any data. In such cases, since load is never called, consume is never called and hence render is never called.

You can
1) Either remove these dataSouce != null checks here and in reinitialize
2) Or add something in the onResume as well to call render with null if datasource is null